manager2/src/app/project/project.component.html

```diff
@@ -92,7 +92,7 @@ <h3>Projects</h3>
             </ng-template>
             <ng-template pTemplate="body" let-project>
               <tr>
-                <td (click)="show_project_users(project)" style="cursor: pointer;"><span class="label label-primary">{{project.id}}</span></td>
+                <td (click)="show_project_users(project); scroll(selectedProjectAnchor)" style="cursor: pointer;"><span class="label label-primary">{{project.id}}</span></td>
```

Comment:
Show project users is asynchronous that s why it is notbyet available, i haven t looked at code but should use await or .then to scroll once function is over

Reply:
It tried wait in but show_project_users ... it said that I can't because it asynchronous :/
How do you know when function is asynchronous?

Reply:
Calls to apis in general are async, many other js funcs are async... (when need callbacks)
I will need to look at specific code parts to see what is possible in this case

If you don t want to bother with this, you could simply put an empty div with an id just above your expected scroll location, and scroll to this div. Projects will appear just below once ready.

Reply:
Heu nope, we should not expect multiple clicks and hope...

Something like

Showprojectusers(..).then(() => {
  Scrollto(...)
})

Should work

Else i will have a look after allhands conf

Reply:
I don't know if it's what you expected but it works!